Apps/Src/ReadTritium.c

```diff
@@ -150,7 +167,7 @@ static void assertTritiumError(tritium_error_code_t motor_err){
 	Error_ReadTritium = (error_code_t)motor_err; // Store error code for inspection info
 	if(motor_err == T_NONE) return; // No error, return
 	
-	if(motor_err != T_HALL_SENSOR_ERR){
+	if(motor_err != T_HALL_SENSOR_ERR && motor_err != T_WATCHDOG_LAST_RESET_ERR){
 		// Assert a nonrecoverable error with no callback function- nonrecoverable will kill the motor and infinite loop
 		assertTaskError(OS_READ_TRITIUM_LOC, Error_ReadTritium, NULL, OPT_LOCK_SCHED, OPT_NONRECOV);
```

Comment:
Also, I should have thought of this when I wrote the assert earlier, but do we want to lock the scheduler for the hall sensor and motor watchdog errors?

Reply:
I don't think we need to. That actually simplifies MotorController_Reset, since we can just use a blocking CANbus_Send to send the reset message